src/game/objects/traps/lava.c

```diff
@@ -114,8 +114,7 @@ void Lava_Control(int16_t fx_num)
             < Room_GetCeiling(floor, fx->pos.x, fx->pos.y, fx->pos.z)) {
         Effect_Kill(fx_num);
     } else if (Lara_IsNearItem(&fx->pos, 200)) {
-        g_LaraItem->hit_points -= LAVA_GLOB_DAMAGE;
-        g_LaraItem->hit_status = 1;
+        Lara_TakeDamage(LAVA_GLOB_DAMAGE, true);
```

Comment:
Glob? I know it's unrelated and this term predates this PR but I was wondering if someone knows where it came from

Reply:
Um you decompiled right lol? Was the name taken from the debug symbols? Glob is a real word and kind of makes sense since for what the lava emitter shoots out. What name would you use?

Reply:
Embers?